### DISCUSSION TITLE
4.2 Geopandas Join - PyGEOS issue

### DISCUSSION THREAD

**Discussion by jenw-ua** [ua-gist604b-f24/syllabus#10] [General]
Hello all,

Working on 4.2. The first line of code is causing an error. I installed each item separately to figure out which item was the problem. All were successful until an error with PyGEOS. After some research, it appears PyGEOS is no longer supported, etc. Here is some documentation on the transition away from PyGEOS (although I could not figure out how to apply it to this module)...and a screenshot of the error. Please advise on how to proceed. Thanks.

https://geopandas.org/en/v1.0.1/docs/user_guide/pygeos_to_shapely.html

<img width="1615" alt="Screenshot 2024-10-29 at 4 28 08 PM" src="https://github.com/user-attachments/assets/a3e1aeca-8cdc-4f55-b64c-29146a29a5ba">

**Comment by travisz09** [comment]
It should work correctly if you just replace pygeos with shapely in the `pip install` command, like so

> pip install geopandas descartes mapclassify folium branca rtree shapely

I had some unrelated questions about lesson 4.2 which I will address in a new thread.

**Reply by jenw-ua** [reply]
Thanks Travis!

**Reply by aaryno** [reply]
Thanks, Travis.
